### ISSUE TITLE
Do not create collection for each NuruMassage scene

### ISSUE THREAD

**Issue by d2dyno** [PAhelper/PhoenixAdult.bundle#304]
Currently, my Plex server 'collections' section is filled with collections, that contains only one scene, for nearly every NuruMassage scene. For a random example, the scene `The Switch - A - Roo`, has a parent collection called `Alex And Cruz`, of which it is the only scene. There are hundreds just like this that I think I will manually have to remove from my server, but perhaps the agent can be updated to not add scenes to their own collection, and instead use the 'Nuru Massage' collection.

**Comment by SgtBatten** [comment]
Please test #305

Some people might want it so left it as a draft. Fixes all the sites you lsited

**Comment by d2dyno** [comment]
It's good of you to think about others :stuck_out_tongue: In my mind, no one wants 1300 'collections' on their collections page, when they really only have maybe 300-400. In my opinion, the 1 scene collections are a bug, not a feature haha.

**Comment by SgtBatten** [comment]
Yeah right, I dont use any sites which do this so i guess i have not seen the issue. I have less than 80 collections easily. so im definitely agreeing with you

**Comment by claygoldfinch** [comment]
Agree with you for the most part, but I think we may need to take a closer look at which sites should and shouldn't be scraped for "DVD names" before removing them completely.

For example:

NetworkGamma:
- Nuru/Fantasy Massage, Girlsway, 21Naturals sites probably shouldn't be scraped for DVD names (as they aren't listed anywhere but in the site code)
- EvilAngel and XEmpire should be scraped for DVD releases

We might want to go through this on a site-by-site basis.

**Comment by SgtBatten** [comment]
Yeah fair enough. As a none user of these sites it's hard to know what peoples collections end up like or what is desired.

The PR gets rid of them completely. Could be done as a list and if the sub site is in list A don't include it.

I'm guessing though that plenty of sub sites have individual titles that are multi-scene and others that are not

**Comment by claygoldfinch** [comment]
Agreed, I think the list method is the best way to go.

There are so many subsites under the NetworkGamma umbrella, it makes it difficult to make changes to a single site without affecting all the sites. I've been playing around with splitting up some of these big networks into smaller, more manageable pieces when major site redesigns occur (ie. [Evil Angel](https://github.com/PAhelper/PhoenixAdult.bundle/issues/248)).

**Comment by d2dyno** [comment]
More sites started having this issue:
- BurningAngel
- DarkX
- DDFBusty
- EroticaX
- FantasyMassage
- Girlsway
- HardX
- ImmoralLive
- LesbianX
- MommyBlowsBest
- OnlyTeenBlowjobs

**Comment by SgtBatten** [comment]
From #420 
> The same happens to me.
> Here are some of the sites that I found this to happen:
> -Girlsway
> -XEmpire
> -Fantasy Massage
> -Jules Jordan
> -LetsDoeIt
> -DDF
> -At least one from Brazzers

**Comment by tritnaha** [comment]
Seeing the same issues here as well unfortunately.
